lib/tasks/migration.rake

```diff
@@ -0,0 +1,26 @@
+namespace :data do
+  namespace :migrate do
+    desc 'Use for unifying new member post types'
+    task new_member_posts: :environment do
+      new_member_post_type_ids = [104, 133, 142, 196, 205, 206, 302, 309, 313, 397, 414, 415, 418,
```

Comment:
I forgot it mentioned, but I think that would be better if we create a query for ids instead of list them.
I mean something like this:
```
names = ['ujonc', 'újonc', 'ùjonc']

new_member_post_types = PostType.where(name: names)
                                .where.not(id: Membership::NEW_MEMBER_ID)
                                .includes(:group)
```
Or
```
name = PostType.arel_table[:pttip_name]

new_member_post_types = PostType.where(name.matches('%jonc%'))
                                .where.not(id: Membership::NEW_MEMBER_ID)
                                .includes(:group)
```
or instead of arel_able (because it doesn't handle the alias)
`.where("pttip_name LIKE '%jonc%'")`

So I think working with raw ids is very dangerous because maybe it works with your local db or prod db but it isn't guarantee that it works for everyone.